pep-0594.rst

```diff
@@ -927,10 +583,6 @@ the ``2to3`` command to transpile Python 2 code to Python 3 code.
 The package is useful for other tasks besides porting code from Python 2 to
 3. For example, `Black`_ uses it for code reformatting.
 
-Module type
```

Comment:
The PEP lists lib2to3 among modules to be kept, but it's been independently scheduled for removal in 3.13 already: https://docs.python.org/3.10/library/2to3.html#module-lib2to3.

Also (speaking as Black maintainer), it's not exactly true that Black depends on lib2to3; instead we use a fork of the module that has been enhanced to support match/case. So from Black's perspective, there's no need to keep it in the stdlib.

Reply:
Also mentioned [here](https://github.com/python/peps/pull/2262#discussion_r791303079), as noted in my review